src/main/java/gtclassic/tile/GTTileFacing.java

```diff
@@ -22,9 +23,7 @@ public GTTileFacing(EnumFacing... validRotations) {
 		super();
 		setFacing(EnumFacing.NORTH);
 		this.validRotations = new LinkedHashSet<>();
-		for (EnumFacing rotation : validRotations) {
-			this.validRotations.add(rotation);
-		}
+		Collections.addAll(this.validRotations, validRotations);
```

Comment:
Use RotationList instead. Its just a Integer and has a lot of helper functions.

Reply:
fixed in ec4dd3fc6fe61fa075c8f98b06a70e0b3471871c